frontend/packages/ceph-storage-plugin/src/selectors/index.ts

```diff
@@ -23,16 +23,14 @@ export const filterCephAlerts = (alerts: Alert[]): Alert[] => {
 export const getCephPVs = (pvsData: K8sResourceKind[] = []): K8sResourceKind[] =>
   pvsData.filter((pv) => {
     return cephStorageProvisioners.some((provisioner: string) =>
-      _.get(pv, 'metadata.annotations["pv.kubernetes.io/provisioned-by"]', '').includes(
-        provisioner,
-      ),
+      (pv?.metadata?.annotations?.['pv.kubernetes.io/provisioned-by'] ?? '').includes(provisioner),
     );
   });
 
-const getPVStorageClass = (pv: K8sResourceKind) => _.get(pv, 'spec.storageClassName');
+const getPVStorageClass = (pv: K8sResourceKind) => pv?.spec?.storageClassName;
 const getStorageClassName = (pvc: K8sResourceKind) =>
-  _.get(pvc, ['metadata', 'annotations', 'volume.beta.kubernetes.io/storage-class']) ||
-  _.get(pvc, 'spec.storageClassName');
+  pvc?.metadata?.annotations?.['volume.beta.kubernetes.io/storage-class'] ||
+  pvc?.spec?.storageClassName;
```

Comment:
nit: first `pvc?.spec?.storageClassName` should be check and if not available then other annotation parameters should be checked.